contrib/kind.sh

```diff
@@ -143,10 +194,18 @@ print_params()
      echo "OVN_EMPTY_LB_EVENTS = $OVN_EMPTY_LB_EVENTS"
      echo "OVN_MULTICAST_ENABLE = $OVN_MULTICAST_ENABLE"
      echo "OVN_IMAGE = $OVN_IMAGE"
+     echo "MASTER_LOG_LEVEL = $MASTER_LOG_LEVEL"
+     echo "NODE_LOG_LEVEL = $NODE_LOG_LEVEL"
+     echo "DBCHECKER_LOG_LEVEL = $DBCHECKER_LOG_LEVEL"
+     echo "OVN_LOG_LEVEL_NORTHD = $OVN_LOG_LEVEL_NORTHD"
+     echo "OVN_LOG_LEVEL_NB = $OVN_LOG_LEVEL_NB"
+     echo "OVN_LOG_LEVEL_SB = $OVN_LOG_LEVEL_SB"
+     echo "OVN_LOG_LEVEL_CONTROLLER = $OVN_LOG_LEVEL_CONTROLLER"
+     echo "OVN_LOG_LEVEL_NBCTLD = $OVN_LOG_LEVEL_NBCTLD"
      echo ""
 }
 
-parse_args $*
+parse_args "$@"
```

Comment:
this is needed to deal with this args `-vconsole:info -vfile:info"` right?

can it break something?

Reply:
Yeah, it's needed to transfer the strings with spaces. 

> can it break something? 
I deployed kind a couple of times, it works correctly unless you put your arguments in quotes.

Reply:
I guess it's ok for me, maybe worth a small comment there but not really a blocker for me

Reply:
A comment was added.